Pidgin.Tests/EnumParserTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Xunit;
+
+namespace Pidgin.Tests;
+
+public class EnumParserTests
```

Comment:
You can just put this in the main test file

Reply:
I hope I understood you correctly